docs/querying/operators.md

```diff
@@ -303,28 +385,6 @@ could write:
 
     limitk(10, http_requests_total)
 
-To deterministically sample approximately 10% of timeseries we could write:
-
-    limit_ratio(0.1, http_requests_total)
-
-Given that `limit_ratio()` implements a deterministic sampling algorithm (based
-on labels' hash), you can get the _complement_ of the above samples, i.e.
-approximately 90%, but precisely those not returned by `limit_ratio(0.1, ...)`
-with:
-
-    limit_ratio(-0.9, http_requests_total)
-
-You can also use this feature to e.g. verify that `avg()` is a representative
```

Comment:
this paragraph is "lost" is that intentional?

Reply:
Yes. (I thought I had mentioned it in the commit description, but I did not. Maybe I squashed things accidentally. I'll amend the commit description accordingly.)

For context: This was way to elaborate for the reference style we use here. Also, a relatively niche operator was now taking more space to explain than the way more common ones. And finally, I think the use case in the example is a bit disputable.